### PR TITLE
Feat/people/person search

### PR DESCRIPTION
**Pull request by eikeland**
Adding FWC-Person-Search element.
Refactoring the controller for realigned task events.
Styling an UX for list-item

### CODE REVIEW

File: packages/person/src/components/search/element.ts [hunk 1]
```diff
@@ -0,0 +1,270 @@
...
+            this.controllers.observer.unobserve(this);
+            this.tasks = {
+              search: new PersonSearchTask(this),
+            };
+          }
+        }
+      },
+    }),
+    element: new PersonSearchController(this),
+  };
```
Comment (odinr) [inline packages/person/src/components/search/element.ts:145]:
```suggestion
  private tasks?: {
-    search: PersonSearchTask;
+    search: new PersonSearchTask(this)
  };
-  @state()
-  protected intersected = false;
-
-  protected controllers = {
-    observer: new IntersectionController(this, {
-      callback: (e) => {
-        if (!this.intersected) {
-          this.intersected = !!e.find((x) => x.isIntersecting);
-          if (this.intersected) {
-            this.controllers.observer.unobserve(this);
-            this.tasks = {
-              search: new PersonSearchTask(this),
-            };
-          }
-        }
-      },
-    }),
-    element: new PersonSearchController(this),
-  };
```

File: packages/person/src/components/search/element.ts [hunk 2]
```diff
@@ -0,0 +1,270 @@
...
+
+  @query('fwc-list')
+  listElement!: ListElement;
+
+  @queryAll('fwc-list-item:not([disabled])')
+  listItems!: Array<ListItemElement<PersonInfo>>;
+
+  private tasks?: {
+    search: PersonSearchTask;
+  };
```
Comment (odinr) [inline packages/person/src/components/search/element.ts:125]:
```suggestion
  private tasks = {
    search: new PersonSearchTask();
  };
```

File: packages/person/src/components/search/element.ts (hunk 1 above)
Comment (odinr) [inline packages/person/src/components/search/element.ts:145]:
```suggestion
```

File: packages/searchable-dropdown/src/dropdown/element.css.ts
Comment (odinr) [inline packages/searchable-dropdown/src/dropdown/element.css.ts:1]: resolve eslint

**Review by odinr — COMMENTED** [review]
sea comments 🌊

File: packages/person/src/components/search/element.ts (hunk 1 above)
Comment (asbjornhaland) [reply packages/person/src/components/search/element.ts]: fixed

File: packages/person/src/components/search/utils.ts
[the remark this answers is not in the document]
Comment (asbjornhaland) [reply packages/person/src/components/search/utils.ts:1]: fixed

**Review by odinr — APPROVED** [review]
works on my machine 🤞🏻

**Pull request merged**